mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/Configuration.java

```diff
@@ -34,6 +36,6 @@
    * to &quot;config&quot;. Configuration name can't be duplicated.
    * Only one config can use the default config name
    */
-  String name() default "";
+  String name() default DEFAULT_CONFIG_NAME;
```

Comment:
@marianogonzalez shouldn't this break the compilation because of kraan's plugin?

Reply:
I'm assuming that compilation was done before sending the PR. Otherwise, I like my pastries with pastelera cream

Reply:
I was aiming more for a bug in the validator 🤔